README.md

```diff
@@ -1,6 +1,7 @@
 # Wire Sysio
```

Comment:
Seems to me we may want some indication this is for development use or experimental use. We typically do not want block producers running nodes in docker because of the poor performance. This is the main reason docker was not provided in Leap. It was removed because it is not a recommended env.